erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -17,6 +17,7 @@ const BaseStack = (specInput) => {
 
   that.pcConfig = {
     iceServers: [],
+    bundlePolicy: 'max-bundle',
```

Comment:
have you tested this property with Chrome/Firefox? My only concern is not to break things that are working with the browsers that we currently support.

Reply:
this breaks p2p firefox, sorry I forgot to update the pr. this must be moved only in EdgeStack, maybe in a separate pr in the future.